rpki-validator/src/main/java/net/ripe/rpki/validator3/storage/xodus/Xodus.java

```diff
@@ -79,7 +80,7 @@ protected synchronized Store meta() {
         AtomicReference<List<Runnable>> afterCommitHooks = new AtomicReference<>(Collections.emptyList());
 
         Environment env = getEnv();
-        T result = env.computeInExclusiveTransaction(txn -> {
+        T result = ForkJoin.blocking(() -> env.computeInExclusiveTransaction(txn -> {
```

Comment:
computeInExclusiveTransaction blocks the whole database, so there can only be one computeInExclusiveTransaction running at every moment. Does it change the logic?

Reply:
There could be many threads waiting for the `computeInExclusiveTransaction` lock. All these threads should not be blocking the fork-join pool, so the logic here should be good.